tenable/io/v3/vm/policies/api.py

```diff
@@ -0,0 +1,395 @@
+                The pagination token to use when requesting the next page of
+                results. This token is presented in the previous response.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
+        Returns:
+            iterable:
+                The iterable that handles the pagination for the job.
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
+                object is instead returned instead of an iterable.
+        Examples:
+            >>> tio.v3.vm.policies.search(
+            ...     filter=filters,
+            ...     fields=field, limit=2)
+        '''
+        iclass = SearchIterator
+
+        if kwargs.get('return_csv', False):
+            iclass = CSVChunkIterator
+
+        return super()._search(
+            iterator_cls=iclass,
+            sort_type=self._sort_type.default,
+            api_path=f'{self._path}/search',
+            resource='policies',
+            **kwargs
+        )
+
+    def templates(self) -> Dict:
+        '''
+        returns a dictionary of the scan policy templates using the
+        format of dict['name'] = 'UUID'.  This is useful for being able to
+        define scan policy templates w/o having to remember the UUID for each
+        individual one.
+        '''
+        # policies = dict()
+        # for item in self._api.v3.editor.search_template('policy'):
+        #     policies[item['name']] = item['uuid']
+        # return policies
+        raise NotImplementedError(
+            'This functionality will be implemented after Editor'
+            ' module is completed'
+            )
+
+    def template_details(self, name: str) -> Dict:
+        '''
+        Calls the editor API and parses the policy template config to return a
+        document that closely matches what the API expects to be POSTed or PUT
+        via the policy create and configure methods.  The compliance audits and
+        credentials are populated into the 'current' sub-document for the
+        relevant resources.
+
+        Args:
+            name (str): The name of the scan template.
+
+        Returns:
+            :obj:`dict`:
+                The policy configuration resource.
+
+        Examples:
+            >>> template = tio.v3.vm.policies.template_details('basic')
+            >>> pprint(template)
+
+        Please note that template_details is reverse-engineered from the
+        responses from the editor API and isn't guaranteed to work.
+        '''
+        # # Get the policy template UUID
```

Comment:
We can keep as of now, with V3 API integration we can enable that part since this branch will merge with [Task/in 581 v3 standardization](https://github.com/dhvanilshah-tenable/pyTenable/pull/104/files#)